modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -850,9 +850,14 @@ Resources:
                       export LC_ALL=en_US.UTF-8
                       export LANG=en_US.UTF-8
 
-                      python3 -m ensurepip --upgrade --default-pip
-                      pip install --index-url="$PYPI_URL" --upgrade pip setuptools boto3
-                      pip install --index-url="$PYPI_URL" --upgrade ${local_WatchmakerPkg}
+                      python3 -m ensurepip
+                      if [[ $(cat /proc/version | grep -o 'el.') == 'el6' ]]
+                      then
+                        python3 -m pip install --index-url="$PYPI_URL" --upgrade pip'<=19.1.*' setuptools boto3
```

Comment:
The only difference here is pip... Can we do something like

```
PIP_PKG=pip
if [[ $(cat /proc/version | grep -o 'el.') == 'el6' ]]
then
  PIP_PKG='pip<=19.1'
fi

python3 -m pip install --index-url="$PYPI_URL" --upgrade "$PIP_PKG" setuptools boto3
```

Reply:
Sure.  I'll make the change.